hack/deployer/README.md

```diff
@@ -1,71 +1,51 @@
 # Deployer
 
-Deployer is the provisioning tool that aims to be the interface to multiple Kubernetes providers.
+Deployer is the provisioning tool that aims to be the interface to multiple Kubernetes providers. Currently, it supports GKE and AKS.
 
-At the moment it support GKE and AKS.
+# Typical usage
```

Comment:
top level heading is already defined at line 1

Reply:
Yes. I'm not sure if this is a big deal, I think it looks much better like that. WDYT?